appliance-root/opt/napp/bin/broker-start

```diff
@@ -48,14 +48,16 @@ if [ -z "`grep circonus-credentials.conf $CONF`" ]; then
 	rm -f $CONF.bak
 fi
 
-# move credentials around change d2b17c3
-O_USER=`$NOITD -u $USER -g $GROUP -x '/noit/circonus/appliance/username' 2>/dev/null | grep '^0:' | cut -c4-`
-O_PASS=`$NOITD -u $USER -g $GROUP -x '/noit/circonus/appliance/password' 2>/dev/null | grep '^0:' | cut -c4-`
-O_INSIDE=`$NOITD -u $USER -g $GROUP -x '/noit/circonus/appliance/inside' 2>/dev/null | grep '^0:' | cut -c4-`
-O_URL=`$NOITD -u $USER -g $GROUP -x '/noit/circonus/appliance/circonus_url' 2>/dev/null | grep '^0:' | cut -c4-`
-if [ -n "$O_USER$O_PASS$O_INSIDE$O_URL" ]; then
-	echo "Copying credentials from noit.conf to circonus-credentials.conf"
-	cat << END_OF_CONF > /opt/noit/prod/etc/circonus-credentials.conf
+if $AWK 'BEGIN{C=0} /<appliance>/{IN=1} /(username|password|inside|circonus_url)/{if(IN==1){C++;}} END{exit(C==0);}' $CONF ; then
```

Comment:
Is there any possibility that any of the terms username, password, inside, or circonus_url could appear outside of an appliance section? We never reset `IN` to 0, so we'll run past the `</appliance>` to the end of the file.

I think it's unlikely but I felt I should mention it.